tests/rules/no-multiple-stores.test.ts

```diff
@@ -11,82 +11,63 @@ ruleTester().run(ruleName, rule, {
     }`,
     `
     export class OneWithVisibilityOk {
-      constructor(private store: Store<{}>){}
+      constructor(private store: Store){}
     }`,
     `
     export class OneWithoutVisibilityOk {
       constructor(store: Store){}
     }`,
     `
     export class OnePlusExtraOk {
-      constructor(private store: Store<{}>, data: Service){}
+      constructor(private store: Store, data: Service){}
     }`,
   ],
   invalid: [
     {
       code: stripIndent`
       export class NotOkNoVisibility {
-        constructor(store: Store<{}>, store2: Store<{}>){}
+        constructor(store: Store, store2: Store){}
       }`,
       errors: [
         {
           messageId,
           line: 2,
-          column: 33,
-          endLine: 2,
-          endColumn: 50,
```

Comment:
Any particular reason to remove these constraints ?

Reply:
Going to rewrite them with a helper function from eslint-etc.
I did the rest already, see https://github.com/timdeschryver/eslint-plugin-ngrx/commit/81a34734dcee5047df0ef56c888c9fbfe511eab3

Reply:
Interesting !